Model/Adminhtml/Page/Source/WordpressPage.php

```diff
@@ -48,8 +48,9 @@ public function toOptionArray()
                     continue;
                 }
                 foreach ($site['data'] as $page) {
```

Comment:
Eigenlijk houd ik er niet zo van om variabelen te declareren en deze maar één keer te gebruiken. Is het mogelijk om de 'date' key in de $page array te veranderen naar 'date_last_modified' en dan date('j M y', strtotime($page['date_last_modified'])) te benutten bij het maken van je 'label'?